ui/components/src/ui-components/tree/controlled/internal/TreeSelectionManager.ts

```diff
@@ -320,10 +315,11 @@ class ItemHandler implements SingleSelectionHandler<string> {
 
   // eslint-disable-next-line @bentley/prefer-get
   public isSelected(): boolean {
-    return this._node.isSelected;
+    // istanbul ignore next
```

Comment:
Is this "ignore next" just because of the "!!" and "node?" ?

Reply:
It's because of `_node?` - `isSelected()` doesn't get called if `this._node` is not set.